docs/user-guide/configuration/externalConfig.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: main
+title: External Config
+category: User Guide
+menu: menu
+toc:
+    - title: External Config
+      url: "#external-config"
+
+---
+# External Config
+External config allows a single parent pipeline to create and manage a list of child pipelines with job configuration from parent's `screwdriver.yaml` and source code from itself.
+
+This feature will benefit users who owning multiple pipelines with same building patterns to manage all pipelines easily.
```

Comment:
This feature will allow users who own~ing~ multiple pipelines with `the` same workflow to manage their pipelines more easily.

Reply:
Perhaps give the meaning that `building multiple source code repositories`  ? Child ones are not pipelines but source code.